app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageController.scala

```diff
@@ -619,6 +620,87 @@ class SingleDisposalsTriageController @Inject() (
       )
   }
 
+  def disposalDateOfShares(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    displayTriagePage(
+      _.fold(_.assetType, c => Some(c.assetType).filterNot(e => e === IndirectDisposal)),
+      _ => routes.SingleDisposalsTriageController.countryOfResidence()
+    )(_ => sharesDisposalDateForm)(
+      _.fold(
+        _.completionDate.map(e => ShareDisposalDate(e.value)),
```

Comment:
even though they contain the same value, it'd be neater to consistently reference the disposal date value over the completion date value to get the disposal date for shares since we always talk about the "disposal" of the shares rather than "completion"